### PR TITLE
feat(docs): wire sync-community-repos.js to sidebars.json to include …

### PR DESCRIPTION
**Pull request by bartolkaruza**
This PR adds shell pages for react-native-community components and API's. Some remaining lean core doc links have been removed from the sidebar.

### CODE REVIEW

File: website/sync-community-repos.js [hunk 1]
```diff
@@ -74,54 +57,233 @@ async function graphqlQuery(after = null) {
...
+      title: 'Webview',
+    },
+    {
+      owner: 'react-native-community',
+      name: 'react-native-cameraroll',
+      type: 'API',
+      title: 'CameraRoll',
+    },
+    {
+      owner: 'harisbaig100',
```
Comment (lucasbento) [inline website/sync-community-repos.js]: What's the `owner` role here? Just asking as some of them have `react-native-community`.

File: website/sync-community-repos.js [hunk 2]
```diff
@@ -74,54 +57,233 @@ async function graphqlQuery(after = null) {
...
-    endCursor = pageInfo.endCursor;
-  } while (endCursor != null);
-
-  return repos;
+  const repos = [
+    {
+      owner: 'react-native-community',
+      name: 'react-native-webview',
+      type: 'Component',
+      title: 'Webview',
```
Comment (lucasbento) [inline website/sync-community-repos.js]: Minor nitpick:

```suggestion
      title: 'WebView',
```

File: website/sync-community-repos.js [hunk 3]
```diff
@@ -136,12 +298,48 @@ async function syncCommunityRepos() {
...
+
+> This ${
+          repo.type
+        } is not a part of React Native Core but maintained by the React Native Community. You can learn more at its [repo](${
+          repo.url
+        }).
+
+Package: \`\`\`${repo.packageName}\`\`\`
+
+Github repository: [${repo.url}](${repo.url})
```
Comment (lucasbento) [inline website/sync-community-repos.js]: Minor nitpick:

```suggestion
GitHub repository: [${repo.url}](${repo.url})
```

File: website/sync-community-repos.js (hunk 1 above)
Comment (bartolkaruza) [reply website/sync-community-repos.js]: that's used to fetch the repo details from GitHub, not for display

**Comment by bartolkaruza** [comment]
@charpeni I've fixed the review comments and tested this locally. I think the PR is ready to be merged

New docs: 
```http://localhost:3000/react-native/docs/next/cameraroll```

Full old docs: 
```http://localhost:3000/react-native/docs/cameraroll```

I don't think we should worry about updating the older versions, we can just have the next version display the shell pages as fallback.

**Comment by charpeni** [comment]
Sorry for the delay, I'll have a look at this soon!

**Comment by bartolkaruza** [comment]
> Sorry for the delay, I'll have a look at this soon!

no worries, let me know if you need any help

**Comment by cpojer** [comment]
Exciting! Could you rebase this?

**Comment by rachelnabors** [comment]
I remember this idea, and it was well-founded. Bringing more community components into the docs would rock. But as I've worked more and more on these docs and with React Native, the more I feel community components should live in a directory or marketplace of their own, as syncing with our current docs infrastructure is cumbersome at best—and there are far too many amazing components for us to feature them all fairly. We would have to play favorites using an opaque evaluation system.

So while I like this idea and this PR, I have to close it for the time being.

I really appreciated your fleshing this idea out and hope we can find a way to better surface community components to React Native users in the near future. Please, keep your good ideas coming, and if you ever want to talk about some, please reach out to me.

Thank you, @bartolkaruza :)

**Pull request closed without merging**